.infrastructure/webpack/make-webpack-config.js

```diff
@@ -1,15 +1,40 @@
 var path = require("path");
 var fs = require("fs");
+var process = require("process");
+var merge = require("lodash/object/merge");
+var transform = require("lodash/object/transform");
+var partial = require("lodash/function/partial");
+var isBoolean = require("lodash/lang/isBoolean");
+var isObject = require("lodash/lang/isObject");
 var yaml = require('js-yaml');
 var webpack = require("webpack");
 var ExtractTextPlugin = require("extract-text-webpack-plugin");
 var ManifestPlugin = require("webpack-manifest-plugin");
 var StatsPlugin = require("stats-webpack-plugin");
 var loadersByExtension = require("./helpers/loadersByExtension");
-var appConfig = yaml.safeLoad(fs.readFileSync('config.yml'));
+
 var ROOT = path.join(__dirname, "..", "..");
 var JS_ROOT = path.join(ROOT, "beavy", "jsbeavy");
+var BEAVY_ENV = process.env.BEAVY_ENV || "DEVELOPMENT";
+
+var appConfig = merge(
+	{},
+	yaml.safeLoad(fs.readFileSync("beavy/config.yml"))[BEAVY_ENV],
+	yaml.safeLoad(fs.readFileSync('config.yml'))
+);
+
+function deepTransform(prefix, result, value, key) {
+	if (isObject(value))
+		transform(value,
+		  				partial(deepTransform, prefix + "_" + key),
+			  			result);
+	if (!isBoolean(value))
+		value = JSON.stringify(value);
```

Comment:
Going for: #58 should do that. I will rebase this PR after merging 58 into master.